protostar/commands/test/fuzzing/hypothesis/runs_counter.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class RunsCounter:
+    """
+    A boxed integer that can be safely shared between Python threads.
+    It is used to count fuzz test runs.
+    """
+
+    budget: int
+    count: int = field(default=0)
+
+    def __next__(self) -> int:
+        self.count += 1
+        return self.count
+
+    @property
+    def balance(self) -> int:
+        return self.budget - self.count
```

Comment:
Is `balance` used anywhere?

Reply:
It's used in available_runs + I have been using this while debugging, that's why I left it. But indeed it might be worth inlining.